tests/test_pr_comments.py

```diff
@@ -215,6 +220,18 @@ def test_has_contractor_comment_no_comments(app, reqctx, requests_mocker):
     assert result is False
 
 
+def test_has_internal_cherry_pick_comment_release_environ_set(app):
+    pr = make_pull_request(user='nedbat')
+    open_edx_release = "Matt's Open edX Release"
+    fun_facts = get_fun_fact_file()
+    question, answer = random.choice(list(fun_facts.items()))
```

Comment:
This is doing too much work for this test, and duplicates lines in the product code.  You can test `github_internal_cherrypick_comment` by passing it literal strings and checking the output.